src/Microsoft.Diagnostics.Runtime/src/DacInterface/SosDac.cs

```diff
@@ -726,7 +726,7 @@ public ulong GetMethodDescFromToken(ulong module, uint token)
         private delegate int DacGetIntPtr(IntPtr self, out IntPtr data);
 
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        private delegate int DacGetUlongWithArg(IntPtr self, ulong arg, out ulong data);
+        private delegate int DacGetUlongWithArg(IntPtr self, ulong arg, out ClrDataAddress data);
 
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
         private delegate int DacGetUlongWithArgs(IntPtr self, ulong arg, uint id, out ulong data);
```

Comment:
Shouldn't this `data` be `ClrDataAddress` as well?

Reply:
Yes it should.

Reply:
There's a few other places this should be changed and we should rename the functions.

Reply:
https://github.com/microsoft/clrmd/pull/448